website/src/components/wordpress.tsx

```diff
@@ -144,7 +144,7 @@ const PullAudio = (props: {
   const doc = data?.document
 
   // Gets the audio recording of this document slice.
-  const docAudio = doc?.audioRecording
+  const docAudio = doc?.ingestedAudioTrack
```

Comment:
The best move here seemed to be to just rely directly on `ingestedAudioTrack` (an alias of the old `audioRecording` field). This code largely seems like it will not work with contributed audio, and per conversation with @GracefulLemming is not in use in production.

Reply:
This looks good for this PR. I think in the future we can also remove audio here completely.